src/uploaders.rs

```diff
@@ -163,13 +163,18 @@ impl Uploader {
         let path = Uploader::readme_path(crate_name, vers);
         let content_length = readme.len() as u64;
         let content = Cursor::new(readme);
+        let mut extra_headers = header::HeaderMap::new();
+        extra_headers.insert(
+            header::CACHE_CONTROL,
+            CACHE_CONTROL_IMMUTABLE.parse().unwrap(),
```

Comment:
The variable `CACHE_CONTROL_IMMUTABLE` is set to `public,max-age=31536000,immutable`, i.e. the files are tagged to never change and to live for one year in the cache. However, the readme files _might_ change, so let's cache them only for a week, and let's not tag them as immutable. (You may want to read the [documentation of the cache-control header](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Cache-Control)).

Reply:
I see, sorry for misunderstanding the original Issue. So I assume the only field that is required is `max-age=<seconds>`?

Reply:
I think you can keep `public` as well. Not sure whether it has any effect when setting `max-age` at the same time, but it certainly doesn't hurt.